src/crawler.ts

```diff
@@ -1974,6 +1971,10 @@ self.__bx_behaviors.selectMainBehavior();
       row.mime = mime;
     }
 
+    if (statusCode !== 200) {
+      row.statusCode = statusCode;
+    }
```

Comment:
I wonder if it's better to write the status code to redis even if it's 200, and simply have the Browsertrix backend return 200 as a default instead of None if it's not set for a page, rather than have all 200s be `None` in the database? Could go either way though honestly.

Reply:
The PageOut now returns 200 if its unset in the db, since previous crawls will have it unset. I guess its a minor space saving, but also backwards compatible with previous usage - eg. if loadState > 0, and no statusCode, assume 200

Reply:
Up to you, but if there ever any other consumers for this data from the crawler/redis, I think it might be clearer to be explicit about the 200 status code rather than relying on knowledge that its absence means 200? Maybe I've just done too many system-to-system migrations where things like this gave me a bit of a headache haha.